core/data/src/main/java/com/maasbodev/core/data/auth/EncryptedSessionStorage.kt

```diff
@@ -0,0 +1,42 @@
+package com.maasbodev.core.data.auth
+
+import android.content.SharedPreferences
+import com.maasbodev.core.domain.AuthInfo
+import com.maasbodev.core.domain.SessionStorage
+import javax.inject.Inject
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+class EncryptedSessionStorage @Inject constructor(
+	private val sharedPreferences: SharedPreferences,
```

Comment:
Where did you provide this SharedPreferences instance? Does Hilt know it?

Reply:
Added it on AppModule